stages/org.osbuild.oscap-remediation

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python3
+"""
+Execute oscap remediation on first boot
+
+The openscap scanner package uses systemd's  `system-update.target` to run an openscap
+remediation at first boot[1]. The `openscap-remediate.service` expects a `/system-update`
+symlink that points to an openscap offline remediation config file. The requirement
+of the config file is to have the basename of `oscap-remediate-offline.conf.sh` and
+must be accessible at boot time.
+
+This stage generates the openscap offline remediation config file with the required
+configurations. A `/system-update` symlink is then created which points to the
+config file.
+
+Once executed, the symlink will be removed to avoid an invocation loop. The oscap
+offline remediation configuration will be left intact.
+
+[1] https://github.com/OpenSCAP/openscap/blob/maint-1.3/docs/manual/manual.adoc#15-scanning-and-remediating-the-system-at-boot-time
+"""
+
+
+import os
+import sys
+
+import osbuild.api
+
+
+SCHEMA = """
+"additionalProperties": false,
+"required": ["profile_id", "datastream"],
+"properties": {
+  "profile_id": {
+    "type": "string",
+    "description": "The scap profile id"
+  },
+  "datastream": {
+    "type": "string",
+    "description": "The path to the datastream file"
+  },
+  "datastream_id": {
+    "type": "string",
+    "description": "The datastream id"
+  },
+  "xccdf_id": {
+    "type": "string",
+    "description": "The xccdf id"
+  },
+  "benchmark_id": {
+    "type": "string",
+    "description": "The benchmark id"
+  },
+  "tailoring": {
+    "type": "string",
+    "description": "The path to the custom tailoring file"
+  },
+  "tailoring_id": {
+    "type": "string",
+    "description": "The tailoring id"
+  },
+  "arf_result": {
+    "type": "string",
+    "description": "Filename and path for saving the arf results"
+  },
+  "html_report": {
+    "type": "string",
+    "description": "Filename and path for saving the html report"
+  },
+  "verbose_log": {
+    "type": "string",
+    "description": "Filename and path for verbose error messages"
+  },
+  "verbose_level": {
+    "type": "string",
+    "enum": ["DEVEL", "INFO", "ERROR", "WARNING"],
+    "description": "The verbosity level for the log messages",
+    "default": "INFO"
```

Comment:
This is not reflected in code. Is this the default of OpenSCAP (if so just remove the `default` here). If you mean that if it is not specified we will set it to `INFO` then we have to adjust the code.

```python3
options.get("verbose_level", "INFO")
```
or if we have generic option handling (see below) then maybe something like:

```
if "verbose_level" not in options:
  options["verbose_level"] = "INFO"
```

Reply:
Oh they have a default set. So I will just remove default from the schema :)